Source/WebKit2/Shared/WebPreferencesDefinitions.h

```diff
@@ -135,7 +135,7 @@
     macro(DOMPasteAllowed, domPasteAllowed, Bool, bool, false) \
     macro(JavaScriptCanAccessClipboard, javaScriptCanAccessClipboard, Bool, bool, false) \
     macro(ShouldPrintBackgrounds, shouldPrintBackgrounds, Bool, bool, DEFAULT_SHOULD_PRINT_BACKGROUNDS) \
-    macro(FullScreenEnabled, fullScreenEnabled, Bool, bool, false) \
+    macro(FullScreenEnabled, fullScreenEnabled, Bool, bool, true) \
```

Comment:
Again, there's WKPreferencesSetFullScreenEnabled, is that instead of this modification.